jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/ProcessServiceImpl.java

```diff
@@ -505,6 +511,32 @@ public Object execute(org.kie.api.runtime.Context context) {
         	disposeRuntimeEngine(manager, engine);
         }
 	}
+
+	private List<String> getActiveBoundaryNodesSignals(ProcessInstance processInstance, Collection<NodeInstance> activeNodes) {
```

Comment:
BTW I noticed that this method is private and it is mixed among public methods. Not sure how is your attitude to that, just sharing my observation.